.env.example

```
@@ -82,9 +82,10 @@ EMBEDDINGS_MODEL_NAME=qwen3-embedding-8b
 
 # Ingest pipeline
 CHUNK_SIZE=2000
+# Permitted range: CHUNK_SIZE 1-100000, CHUNK_OVERLAP 0 to CHUNK_SIZE/2
 CHUNK_OVERLAP=400
 BATCH_SIZE=20
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BATCH_SIZE key should go before the CHUNK_OVERLAP key

(UnorderedKey)